homeassistant/components/sector_alarm/sensor.py

```diff
@@ -0,0 +1,61 @@
+import logging
+from datetime import timedelta
+from homeassistant.components.sector_alarm import DOMAIN as SECTOR_DOMAIN
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+from homeassistant.const import TEMP_CELSIUS
+
+UPDATE_INTERVAL = timedelta(minutes=5)
+
+DEPENDENCIES = ['sector_alarm']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config, async_add_entities,
+                               discovery_info=None):
+    """Initialition of the platform."""
+    sector_connection = hass.data.get(SECTOR_DOMAIN)
+
+    temps = sector_connection.GetTemps()
```

Comment:
This makes a request, which isn't allowed in a coroutine. Change `async_setup_platform` to `setup_platform`.

Reply:
To clarify, regular sync I/O isn't allowed in the event loop, ie in coroutines.